package.json

```diff
@@ -121,7 +121,7 @@
     "vue-focus": "^2.1.0",
     "vue-infinite-loading": "^2.2.1",
     "vue-infinite-scroll": "^2.0.0",
-    "vue-multiselect": "2.0.2",
+    "vue-multiselect": "^2.1.4",
```

Comment:
Package was updated since `singleLabel` slot is not available `@2.0.2`. As you asked, the problem exists with components with enabled `search` feature (placeholder is permanently displayed). As a quick "workaround" I'll set `searchable` option default value to `false`, but as we discussed, I'am going to report an issue to [`vue-multiselect`](https://github.com/shentao/vue-multiselect).
Other than that, I don't see any breaking change.